promotion/aggregation.go

```diff
@@ -0,0 +1,14 @@
+package promotion
+
+import (
+	"time"
+
+	"github.com/shopspring/decimal"
+)
+
+// Aggregation outlines the state of a wallet's claims
+type Aggregation struct {
```

Comment:
I think `Aggregation` is a little bit vague. what about `ClaimSummary`? We could also probably drop `type` since it's implied by context